### ISSUE TITLE
Compilaton warning in Deque

### ISSUE THREAD

**Issue by caxapexac** [DanielChappuis/reactphysics3d#278]
Deque.h::118

Warning about not all execution paths return the value 
Assert doesn't counts (MSVC c++20)

```cpp
...
            else {
                assert(false);
            }
        }
...
```

Need to add something like `return mChunks[0][0];`

**Comment by mrakh** [comment]
Already addressed in pull request #258.

**Comment by caxapexac** [comment]
Oh thanks
